### PR TITLE
stubtest: speed up tests by a lot

### PR DESCRIPTION
**Pull request by hauntsaninja**
I probably should have done this earlier...

### CODE REVIEW

File: mypy/test/teststubtest.py [hunk 1]
```diff
@@ -27,11 +27,41 @@ def use_tmp_dir() -> Iterator[None]:
...
+class int: ...
+class float: ...
+class bool(int): ...
+class str: ...
+class bytes: ...
+
+def property(f: T) -> T: ...
+def classmethod(f: T) -> T: ...
+def staticmethod(f: T) -> T: ...
+"""
```
Comment (ethanhs) [inline mypy/test/teststubtest.py:57]: Flake8 seems to want 2 lines here.
```suggestion
"""

```

File: mypy/test/teststubtest.py [hunk 2]
```diff
@@ -27,11 +27,41 @@ def use_tmp_dir() -> Iterator[None]:
 
 TEST_MODULE_NAME = "test_module"
 
+stubtest_builtins_stub = """
```
Comment (ethanhs) [inline mypy/test/teststubtest.py:30]: Is this currently read off of disk?

**Review by ethanhs — COMMENTED** [review]
This looks really great! Thank you for doing this.

File: mypy/test/teststubtest.py (hunk 2 above)
Comment (hauntsaninja) [reply mypy/test/teststubtest.py:30]: Yup!

**Pull request merged**